src/Components/Header/Nav.jsx

```diff
@@ -84,6 +83,13 @@ export const Nav = () => {
                         {/* <span class="material-icons icon-theme text-3xl">{theme ? "dark_mode" : "light_mode"}</span> */}
                     </button>
                 </li>
+
+                <li className={`${loggedIn ? "flex flex-column flex-justify-center flex-align-center" : "display-none" }`}>
+                    <Link className="flex flex-column" to="/login">
+                        <span className="material-icons text-2xl nav-links nav-profile" onClick={()=> logout()}>logout</span>
```

Comment:
minor suggestion - just `{logout}` would also work in the `onClick` handler. Although it's the same thing, it takes less code.